docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,286 @@
+# Add expiration conditions to TODO comments
+
+Makes it possible to pass arguments to TODO, FIXME and XXX comments to trigger ESLint.
+
+From [ESLint's documentation](https://eslint.org/docs/rules/no-warning-comments):
+
+> Developers often add comments to code which is not complete or needs review.
+
+TODO comments are useful when a piece of code needs some work.
+Unfortunately these can be easily forgotten as it's easy to forget to
+track them, leaving dangling tasks to be found at later random moments.
+
+With this rule a TODO can have right from the beginning a condition
+to define it's lifespan. When the condition is met, ESLint will take care
+of reporting that there's work to be done.
+
+This rule also define that **there must be no TODO comment without conditions** so that you should take more care before simply adding
```

Comment:
```suggestion
This rule also defines that **there must be no TODO comment without conditions** so that you should take more care before simply adding
```

Reply:
Done